ui/components/ui/typography/typography.stories.js

```diff
@@ -1,52 +1,69 @@
 import React from 'react';
-import { number, select, text } from '@storybook/addon-knobs';
 import {
   COLORS,
   FONT_WEIGHT,
   TEXT_ALIGN,
   TYPOGRAPHY,
 } from '../../../helpers/constants/design-system';
+import README from './README.mdx';
 import Typography from '.';
 
 export default {
   title: 'Components/UI/Typography',
   id: __filename,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    color: {
+      control: { type: 'select' },
+      options: COLORS,
+      defaultValue: COLORS.BLACK,
+    },
+    align: {
+      control: { type: 'select' },
+      options: TEXT_ALIGN,
+      defaultValue: TEXT_ALIGN.LEFT,
+    },
+    fontWeight: {
+      control: { type: 'select' },
+      options: FONT_WEIGHT,
+      defaultValue: FONT_WEIGHT.NORMAL,
+    },
+    spacing: {
+      control: { type: 'range', min: 1, max: 8 },
+      defaultValue: 1,
+    },
```

Comment:
Looks like although this knob was in the previous story it doesn't work. The spacing for Typography is similar to the `Box` component. I think we can remove it for now.